core/templates/pages/learner-dashboard-page/goals-tab.component.spec.ts

```diff
@@ -31,6 +31,7 @@ import { GoalsTabComponent } from './goals-tab.component';
 import { EventEmitter, NO_ERRORS_SCHEMA } from '@angular/core';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { WindowDimensionsService } from 'services/contextual/window-dimensions.service';
+import { i } from 'mathjs';
```

Comment:
Same here.

Reply:
same as above

Reply:
As stated, this isn't a simple variable. This is a constant representing the square root of `-1`. It has no reason to be imported.